compiler/crates/relay-typegen/src/visit.rs

```diff
@@ -529,6 +539,13 @@ fn get_resolver_arguments(
                 optional: false,
             });
         }
+    } else if resolver_metadata.live {
```

Comment:
I think it's possible to have a resolver that:

1. Does not define a rootFragment
2. Is not `@live`
3. Reads arguments

In this case, we probably still need some placeholder parent object/key argument.

So maybe we just do `else` here instead of `else if resolver_metadata.live`.

Either way, let's make sure we have a test case for that combination.